pkg/config/components/OTelGRPCReceiver.yaml

```diff
@@ -0,0 +1,39 @@
+name: OTel_GRPC_Receiver
+kind: OTelGRPCReceiver
+summary: Receives OpenTelemetry gRPC traces
+description: |
+  Imports telemetry to Honeycomb from OpenTelemetry gRPC traces (not HTTP).
+ports:
```

Comment:
any thoughts on renaming this to `io` or `connections`.... i dont have any other alternatives... i thought of `connectors`, that might be acceptable if the collector concept is never bubbled up to this level